HPC/README.md

```diff
@@ -184,6 +186,11 @@ Thread model: posix
 InstalledDir: /shared/arm/arm-linux-compiler-23.04_Ubuntu-20.04/bin
 ```
 
+### Storage
+Some HPC applications require significant amounts of file I/O, however HPC instance types (Graviton instances included) don't have significant amounts of local storage, and have extremely limited EBS bandwidth and IOPS.  Relying on EBS on each node can cause surprise slow-downs when the instance runs out of EBS burst credits.  This is one reason we don't recommend using an hpc7g (or any HPC instance type) for headnodes, since the headnode performs additional I/O as the scheduler.  For these reasons the following recommendations are made:
+ - Use FSx for Lustre to serve files to compute nodes.  FSx for Lustre file systems can be configured in a variety of sizes and throughputs to meet your specific needs
+ - Headnodes should be compute-optimized instances (rather than HPC-optimized instances) to ensure sufficient EBS bandwidth to support typical user access patterns as well as SLURM scheduler operation.
```

Comment:
It's worthwhile to mentioned fsx is included in the recommended cluster setup.

Reply:
Should they use C7gn, C7g, or something else? Or is it vague on purpose?